frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -974,7 +974,10 @@ function ReaderHighlight:exportToDocument(page, item)
 end
 
 function ReaderHighlight:addNote()
-    self:handleEvent(Event:new("addNote"))
+    self:handleEvent(Event:new("AddNote"))
```

Comment:
self **.ui** :handleEvent - as done at the top of ReaderHighlight:saveHighlight()
We want `self.ui` to handleEvent, as it will propagate it to all the things registered to self.ui, which includes the plugins.